dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml

```diff
@@ -0,0 +1,27 @@
+title: AWS Cloudwatch metrics
+release: ga
+type: metrics
+
+# List of supported streams. There can be multiple stream definitions for different input types.
+streams:
+  - input: aws/metrics
+    title: AWS cloudwatch metrics
+    description: AWS cloudwatch metrics
+    vars:
+      - name: period
+        description: Cloudwatch metrics.
+        type: period
+        default: 5m
+
+# TODO: The configs for the cloudwatch are pretty complex, arrays of configs.
+
+
+#  metrics:
+#    - namespace: AWS/EC2
+#      #name: ["CPUUtilization", "DiskWriteOps"]
+#      tags.resource_type_filter: ec2:instance
+#      #dimensions:
```

Comment:
@jen-huang This is the part where I think it becomes tricky to build a UI around it. We have an array with values inside. I think here we should allow the user to add his own yaml (in the future) for this property. Something like offer a textarea?

Reply:
Let's chat about this as I'm not 100% clear on this

Reply:
Lets try to sync up later today on it.